addons/mail/static/src/models/thread/thread.js

```diff
@@ -1206,8 +1218,12 @@ function factory(dependencies) {
                 'messagingCurrentPartner',
             ],
         }),
+        /**
+         * see pinnedStatus
+         */
```

Comment:
Put a comment, it's annoying to look at docstring of `pinnedStatus` AND the implementation of `_computeIsPinned` to know what this is.